scripts/qa_crowbarsetup.sh

```diff
@@ -3341,8 +3341,8 @@ function oncontroller_run_tempest
         local opts=$tempestoptions
         if iscloudver 7plus; then
             ### backward compatibility, remove
-            opts=${opts/-s/--smoke}
-            opts=${opts/-t/--serial}
+            opts=$( echo "$opts" | sed 's/-s\( \|$\)/--smoke\1/'  )
+            opts=$( echo "$opts" | sed 's/-t\( \|$\)/--serial\1/' )
```

Comment:
Well I would have done if there had been somewhere to check for known failures.  That is **exactly** what I created the failure groups on the Jenkins Green board for, so it makes me glad that I resurrected them today.